app/src/main/java/org/luxoft/sdl_core/SdlLauncherService.java

```diff
@@ -85,7 +86,7 @@ public void onDestroy() {
         try {
             StopSDLNative();
             if(sdl_thread_.isAlive()) {
-                sdl_thread_.join();
+                sdl_thread_.join(SDL_JOIN_WAIT);
```

Comment:
@VadymLuchko do we really need this timeout if SDL is now shuts down properly?

Reply:
@AKalinich-Luxoft 
Seems sdl_thread_ is blocked when join() without waiting.

I'm not sure of the reason, but I noticed that when I using debug mode, the thread stops correctly.
But if I just run app and join() without waiting, then we get a hang in this place.

Reply:
@VadymLuchko suggest increasing `SDL_JOIN_WAIT` to 5 sec in this case